microreactors/hpmr/steady/thermal_ss.i

```diff
@@ -0,0 +1,445 @@
+    block = ${moderator_blocks}
+    prop_names = 'D Q'
+    prop_values = '1 ${fparse 5.3e3 * qvalue_multiplier}'
+  []
+[]
+
+[Functions]
+[]
+
+[Executioner]
+  type = Transient
+  petsc_options_iname = '-pc_type -pc_hypre_type -ksp_gmres_restart'
+  petsc_options_value = 'hypre boomeramg 300'
+  line_search = 'none'
+  automatic_scaling = true
+
+  l_tol = 1e-02
+  nl_abs_tol = 1e-7
+  nl_rel_tol = 1e-8
+
+  l_max_its = 50
+  nl_max_its = 25
+
+  start_time = 0
+  end_time = 3e4
+
+  [TimeStepper]
+    # type = SolutionTimeAdaptiveDT
+    type = IterationAdaptiveDT
+    growth_factor = 1.5
+    dt = 0.01 # the initial timestep should be very small to avoid discrepancy due to lagging at the beginning of each new Richardson iteration
+  []
+  dtmax = 5e3 # convergence of the global energy balance has less to do with the dt and end_time and more with the actual number of time steps!! (at least after the internal balance in both models is really close to 0)
+  dtmin = 1e-3
+[]
+
+[UserObjects]
+  [average_Twall_hp]
+    type = NearestPointLayeredSideAverage
+    boundary = 'hp'
+    variable = Tsolid
+    direction = z
+    bounds = '0.2 2.0' # evaporator section - FIXME: could add more layers but delta T less than 50K (do it after seeing if it messes with Sockeye)
+    points_file = '../mesh/twelfth_core_hp_centers.txt' # assembly centers
+    execute_on = 'initial timestep_end'
+  []
+[]
+
+[MultiApps]
+  [sockeye]
+    type = TransientMultiApp
+    positions_file = '../mesh/twelfth_core_hp_centers.txt' # 101 HPs
+    app_type = SockeyeApp
+    input_files = 'heatpipe_vapor_only.i'
+    execute_on = 'timestep_end'
+    max_procs_per_app = 1
+    sub_cycling = true
+    max_failures = 1000 # number of times the sub-app is allowed to cut its timesteps before the master-app cuts it
+    output_in_position = true
+    cli_args = 'weight=0.5 weight=1 weight=1 weight=1 weight=1 weight=0.5 weight=1 weight=1 weight=1 weight=1 weight=0.5 weight=1 weight=1 weight=1 weight=1
+                weight=0.5 weight=1 weight=1 weight=1 weight=1 weight=0.5 weight=1 weight=1 weight=1 weight=1 weight=0.5 weight=1 weight=1 weight=1 weight=0.5
+                weight=1 weight=1 weight=0.5 weight=1 weight=0.5 weight=0.5 weight=1 weight=1 weight=0.5 weight=1 weight=1 weight=1 weight=1 weight=0.5 weight=1
+                weight=1 weight=1 weight=1 weight=1 weight=0.5 weight=1 weight=1 weight=1 weight=1 weight=1 weight=1 weight=0.5 weight=1 weight=1 weight=1
+                weight=1 weight=1 weight=1 weight=1 weight=1 weight=1 weight=1 weight=1 weight=0.5 weight=1 weight=1 weight=0.5 weight=1 weight=1 weight=1
+                weight=1 weight=0.5 weight=1 weight=1 weight=1 weight=1 weight=1 weight=0.5 weight=1 weight=1 weight=1 weight=1 weight=1 weight=1 weight=0.5
+                weight=1 weight=1 weight=1 weight=1 weight=1 weight=1 weight=1 weight=1 weight=1 weight=1 weight=1'
+  []
+[]
+
+[Transfers]
+  [to_sockeye_twall]
+    type = MultiAppUserObjectTransfer
```

Comment:
```suggestion
    type = MultiAppGeneralFieldUserObjectTransfer
```

Reply:
Not yet addressed. This change is not always straightforward due to the bounding box that needs to be adjusted.

Reply:
if you dont want to use bounding boxes there is an option now. Use "greedy_search = true"